logagg_fs/mirrorfs.py

```diff
@@ -0,0 +1,306 @@
+import os, sys
+from errno import *
+from stat import *
+import fcntl
+
+try:
+    import _find_fuse_parts
```

Comment:
is there a case where this import file will be present because there is no file with the name in this PR.